pmm-app/src/shared/components/Form/InputLabel/InputLabel.tsx

```diff
@@ -0,0 +1,15 @@
+import React, { FC, forwardRef } from 'react';
+import { cx } from 'emotion';
+import { useTheme } from '@grafana/ui';
+import { getStyles } from './InputLabel.style';
+
+type InputLabelProps = React.LabelHTMLAttributes<HTMLLabelElement>;
+
+export const InputLabel: FC<InputLabelProps> = forwardRef<HTMLLabelElement, InputLabelProps>(({ children, className = '', ...props }, ref) => {
```

Comment:
You can leave className without the default - `undefined` would work as well.

Reply:
Right, done